x-pack/test_serverless/api_integration/test_suites/observability/threshold_rule.ts

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { cleanup, generate } from '@kbn/infra-forge';
+import { Aggregators, Comparator } from '@kbn/observability-plugin/common/threshold_rule/types';
+import { FIRED_ACTIONS_ID } from '@kbn/observability-plugin/server/lib/rules/threshold/threshold_executor';
+import expect from '@kbn/expect';
+import { OBSERVABILITY_THRESHOLD_RULE_TYPE_ID } from '@kbn/observability-plugin/common/constants';
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { createIndexConnector, createRule } from './helpers/alerting_api_helper';
+import { createDataView, deleteDataView } from './helpers/data_view';
+import { waitForAlertInIndex, waitForRuleStatus } from './helpers/alerting_wait_for_helpers';
+
+export default function ({ getService }: FtrProviderContext) {
+  const esClient = getService('es');
+  const supertest = getService('supertest');
+  const esDeleteAllIndices = getService('esDeleteAllIndices');
+  const logger = getService('log');
+
+  describe('Threshold rule', () => {
+    const THRESHOLD_RULE_ALERT_INDEX = '.alerts-observability.threshold.alerts-default';
+    const ALERT_ACTION_INDEX = 'alert-action-threshold';
+    const DATA_VIEW_ID = 'data-view-id';
+    let infraDataIndex: string;
+    let actionId: string;
+    let ruleId: string;
+
+    before(async () => {
+      infraDataIndex = await generate({ esClient, lookback: 'now-15m', logger });
+      await createDataView({
+        supertest,
+        name: 'metrics-fake_hosts',
+        id: DATA_VIEW_ID,
+        title: 'metrics-fake_hosts',
+      });
+    });
+
+    after(async () => {
+      await supertest.delete(`/api/alerting/rule/${ruleId}`).set('kbn-xsrf', 'foo');
+      await supertest.delete(`/api/actions/connector/${actionId}`).set('kbn-xsrf', 'foo');
+      await esClient.deleteByQuery({
+        index: THRESHOLD_RULE_ALERT_INDEX,
+        query: { term: { 'kibana.alert.rule.uuid': ruleId } },
+      });
+      await esClient.deleteByQuery({
+        index: '.kibana-event-log-*',
+        query: { term: { 'kibana.alert.rule.consumer': 'alerts' } },
+      });
+      await deleteDataView({
+        supertest,
+        id: DATA_VIEW_ID,
+      });
+      await esDeleteAllIndices([ALERT_ACTION_INDEX, infraDataIndex]);
+      await esClient.deleteByQuery({
+        index: THRESHOLD_RULE_ALERT_INDEX,
+        query: { term: { 'kibana.alert.rule.uuid': ruleId } },
+      });
```

Comment:
This seems to be duplicate.

Reply:
@benakansara, one is for the Kibana log that is related to the rule, and the other one is for the rule's alerts. So not duplication